components/observatorium.libsonnet

```diff
@@ -9,18 +9,23 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       'app.kubernetes.io/part-of': 'observatorium',
       'app.kubernetes.io/instance': obs.config.name,
     },
+    replicaLabels:: ['prometheus_replica', 'rule_replica', 'replica'],
```

Comment:
This is vertical compaction, so.... `prometheus_replica` should not be touched IMO.

Reply:
Same with rule replica

Reply:
BTW this looks like really offline dedup will help to reduce data even more 2x... right? @brancz @squat

Reply:
as 2 replicas of Prom are pushing

Reply:
> BTW this tells me that really proper offline dedup will help to reduce data even more 2x... right?

Not yet but eventually. It's specific to our case though, because we don't write directly with remote write, but scrape 1 of the prometheus replicas federation endpoint and convert that to remote write, so we really just have 1 (almost always).

Still

> This is vertical compaction, so.... prometheus_replica should not be touched IMO.

this is not an opinion, this is critical, especially because we _have_ pushed data via both remote write replicas before.